### PR TITLE
Set last_activity_at when status is updated (Closes #747)

### LINKED ISSUE #747: Update conversation `last_activity_at` when opened/closed
**Problem**
If a conversation is closed or reopened, we should update the `last_activity_at` field so that the conversations are sorted properly in the dashboard.

**Testing**
Please write automated tests and test manually.

**Questions, or need help getting started?**
Feel free to ask below, or [ping us on Slack](https://join.slack.com/t/papercups-io/shared_invite/zt-h0c3fxmd-hZi1Zp8~D61S6GD16aMqmg) :)

(You can also check out our [CONTRIBUTING.md](https://github.com/papercups-io/papercups/blob/master/CONTRIBUTING.md))

### PR DESCRIPTION
**Pull request by a8t**
### Description

Sets last_activtiy_at 
Adds test.

### Issue

https://github.com/papercups-io/papercups/issues/747

## Checklist

- [ ] Everything passes when running `mix test`
- [ ] Ran `mix format`
- [ ] No frontend compilation warnings

### CODE REVIEW

**Comment by a8t** [comment]
@reichert621

File: lib/chat_api/conversations/conversation.ex [hunk 1]
```diff
@@ -87,15 +87,23 @@ defmodule ChatApi.Conversations.Conversation do
     |> cast(attrs, [:last_activity_at])
   end
 
-  defp put_closed_at(%Ecto.Changeset{valid?: true, changes: %{status: status}} = changeset) do
+  defp put_closed_and_last_activity_at(
+         %Ecto.Changeset{valid?: true, changes: %{status: status}} = changeset
+       ) do
+    now = DateTime.utc_now() |> DateTime.truncate(:second)
```
Comment (reichert621) [inline lib/chat_api/conversations/conversation.ex:93]: hmm i know we do this in some other places but i forget why we truncate by `:second` here... do you happen to know?

Comment (a8t) [reply lib/chat_api/conversations/conversation.ex:93]: Nope, I was curious too. It was there and I left it in. I can remove it – I don't really see any reason for it?

Comment (reichert621) [reply lib/chat_api/conversations/conversation.ex:93]: yeah maybe just remove it and see if it breaks anything 🤷 but if it doesn't, let's avoid truncating

Comment (a8t) [reply lib/chat_api/conversations/conversation.ex:93]: oh lol this is why 

image

Comment (reichert621) [reply lib/chat_api/conversations/conversation.ex:93]: ahhh good to know!

**Review by reichert621 — APPROVED** [review]
:+1:

**Pull request merged**